src/components/Modals/AddContactModal.jsx

```diff
@@ -103,16 +58,9 @@ const AddContactModal = ({ addContact, showAddContactModal, setShowAddContactMod
       webId: event.target.addWebId.value
     };
 
-    notifyStartSubmission(userObject, state, dispatch);
     try {
       await addContact(userObject);
     } finally {
-      runNotification(
-        `"${userObject.givenName} ${userObject.familyName}" added to client list`,
-        5,
-        state,
-        dispatch
-      );
       addNotification(
```

Comment:
I don't think this whole try/finally block is correct. 

- I believe the successful `addNotification()` needs to be in the `try` block after the `await`.
- We need a `catch` block.
- This `setTimeout()` clears the inputs after 2 seconds and closes modal. <-- I'm not sure the others have this. Or is necessary. I don't know, but we should figure it out and make it consistent with rest, whichever way it needs to be.

My personal thoughts are clearing the inputs is great (if it's necessary), and maybe modal should only close on a `success`? So, if `error` the user can try again easier?

Reply:
Personally, I still see this as "implementing the new messaging system" - I should've caught in on last PR

Reply:
Made these changes in the try catch block.